cmd/common/image/verify.go

```diff
@@ -0,0 +1,69 @@
+package image
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/distribution/reference"
+	"github.com/spf13/cobra"
+	"oras.land/oras-go/v2/registry/remote"
+
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/signature"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/signature/cosign"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/signature/notation"
+)
+
+func NewVerifyCmd() *cobra.Command {
+	var publicKey []string
```

Comment:
Since you have several public keys:

```suggestion
	var publicKeys []string
```